assets/scss/index.scss

```diff
@@ -38,6 +38,14 @@
   display: block !important;
 }
 
+.home__sidebar {
+  display: none;
+
+  @media only screen and (min-width: 992px) {
+    display: none !important;
+  }
+}
```

Comment:
I don't know if this is the *best* place for this styling, but given the `home.scss` file comes before the following, I wasn't sure where to put this:

```
.show {
  display: block !important
}
```

We could also forgo showing the sidebar completely on the homepage, but if we do that we should remove the button in the header in mobile.
<img width="832" alt="Screenshot 2024-10-09 at 7 15 33 PM" src="https://github.com/user-attachments/assets/2201a0af-0d4c-4217-8eb7-dce9750c83d8">